src/glide/docs/contributing.md

```diff
@@ -8,6 +8,8 @@ You must also verify that your system has the dependencies that Firefox requires
 - [Linux](https://firefox-source-docs.mozilla.org/setup/linux_build.html)
 - Windows is not officially supported, but [WSL](https://firefox-source-docs.mozilla.org/setup/windows_wsl_build.html) may work
 
+In additon to the Firefox requirements, make sure you have [Node](https://nodejs.org/en/blog/release/v24.0.0) v24 and [pnpm](https://pnpm.io/installation) installed.
```

Comment:
This is a little duplicative with the very first sentence on this page, I think it'd make sense to update that instead? e.g. "... you must have Node.js v24 and pnpm..."

I like updating pnpm to link directly to the install page, that's a nice touch :)